tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv

```diff
@@ -197,4 +197,42 @@
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2090","The ValidateNotNullOrEmpty attribute has been added to parameter 'AuthorizationRuleName' for cmdlet 'Get-AzureRmEventHubAuthorizationRule'.","Remove the ValidateNotNullOrEmpty attribute from parameter 'AuthorizationRuleName'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'NamespaceName' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'NamespaceName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureRmEventHubAuthorizationRule","Get-AzureRmEventHubAuthorizationRule","0","2100","The parameter 'EventHubName' in cmdlet 'Get-AzureRmEventHubAuthorizationRule' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'EventHubName' back to the parameter set '__AllParameterSets'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureEventHubKey","Get-AzureRmEventHubKey","0","1020","The cmdlet 'Get-AzureRmEventHubKey' no longer has output type 'Microsoft.Azure.Commands.EventHub.Models.ListKeysAttributes'.","Make cmdlet 'Get-AzureRmEventHubKey' return type 'Microsoft.Azure.Commands.EventHub.Models.ListKeysAttributes'."
+"D:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.EventHub\Microsoft.Azure.Commands.EventHub.dll","Microsoft.Azure.Commands.EventHub.Commands.EventHub.GetAzureEventHubKey","Get-AzureRmEventHubKey","0","2100","The parameter 'NamespaceName' in cmdlet 'Get-AzureRmEventHubKey' is no longer in the parameter set '__AllParameterSets'.","Add parameter 'NamespaceName' back to the parameter set '__AllParameterSets'."
```

Comment:
Many of these look like actual breaking changes, which won't be accepted (for example, adding validation that wasn't ehere before)